woof-code/rootfs-skeleton/root/.xinitrc

```diff
@@ -41,14 +41,20 @@ fi
 
 xhost +local: 2>/dev/null
 
+if [ "$GTK_MODULES" != "" ] && [ "$(echo "$GTK_MODULES" | grep "canberra\-gtk\-module")" == "" ];then 
```

Comment:
Is sound enabled by default, or ptheme needs to modified to enable sound (https://wiki.archlinux.org/index.php/Libcanberra#Configuration)?

Reply:
Nope it is not enabled by default. But it depends on the Desktop Environment or user if they want sounds.